src/codemagic/tools/app_store_connect/action_groups/certificates_action_group.py

```diff
@@ -0,0 +1,228 @@
+            self._save_certificate(certificate, private_key, p12_container_password, p12_container_save_path)
+        return certificate
+
+    @cli.action(
+        "get",
+        CertificateArgument.CERTIFICATE_RESOURCE_ID,
+        CertificateArgument.PRIVATE_KEY,
+        CertificateArgument.PRIVATE_KEY_PASSWORD,
+        CertificateArgument.P12_CONTAINER_PASSWORD,
+        CertificateArgument.P12_CONTAINER_SAVE_PATH,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.CERTIFICATES,
+        deprecation_info=cli.ActionDeprecationInfo("get-certificate", "0.49.0"),
+    )
+    def get_certificate(
+        self,
+        certificate_resource_id: ResourceId,
+        certificate_key: Optional[Union[PrivateKey, Types.CertificateKeyArgument]] = None,
+        certificate_key_password: Optional[Types.CertificateKeyPasswordArgument] = None,
+        p12_container_password: str = "",
+        p12_container_save_path: Optional[pathlib.Path] = None,
+        save: bool = False,
+        should_print: bool = True,
+    ) -> SigningCertificate:
+        """
+        Get specified Signing Certificate from Apple Developer portal
+        """
+
+        private_key = self._get_certificate_key(certificate_key, certificate_key_password)
+        if save and not private_key:
+            raise AppStoreConnectError("Cannot save resource without certificate private key")
+
+        certificate = self._get_resource(certificate_resource_id, self.api_client.signing_certificates, should_print)
+
+        if save:
+            self._save_certificate(
+                certificate,
+                cast(PrivateKey, private_key),
+                p12_container_password,
+                p12_container_save_path,
+            )
+        return certificate
+
+    @cli.action(
+        "delete",
+        CertificateArgument.CERTIFICATE_RESOURCE_ID,
+        CommonArgument.IGNORE_NOT_FOUND,
+        action_group=AppStoreConnectActionGroup.CERTIFICATES,
+        deprecation_info=cli.ActionDeprecationInfo("delete-certificate", "0.49.0"),
+    )
+    def delete_certificate(
+        self,
+        certificate_resource_id: ResourceId,
+        ignore_not_found: bool = False,
+    ) -> None:
+        """
+        Delete specified Signing Certificate from Apple Developer portal
+        """
+
+        self._delete_resource(self.api_client.signing_certificates, certificate_resource_id, ignore_not_found)
+
+    @cli.action(
+        "list",
+        CertificateArgument.CERTIFICATE_TYPES_OPTIONAL,
+        CertificateArgument.PROFILE_TYPE_OPTIONAL,
+        CertificateArgument.DISPLAY_NAME,
+        CertificateArgument.PRIVATE_KEY,
+        CertificateArgument.PRIVATE_KEY_PASSWORD,
+        CertificateArgument.P12_CONTAINER_PASSWORD,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.CERTIFICATES,
+        deprecation_info=cli.ActionDeprecationInfo("list-certificates", "0.49.0"),
+    )
+    def list_certificates(
```

Comment:
Used to be defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L565-L625